### PR TITLE
Pull Lane's updated webpage into UAH-ACM/website

### PR DESCRIPTION
**Pull request by computablee**
# Pull Request Name / Title
Did some updates to the philosophy section (made it into more of a logic section) and did some general updates to the rest of the site. no css changes, just reflecting where i am in my education right now

# Category
- [x] One-Month Project Submission
- [ ] Bug Fix

# PR Checklist
Complete this checklist before opening your 
- [ ] If the PR is a bug fix, the issue is linked in the PR Content
- [x] If this PR is a One-Month submission, include if you want your site to be listed publicly on our website.
- [ ] If this PR is a work in progress, it's name starts with `WIP:`. Example: `WIP: Nathan Solomon One-Month Submission`

### CODE REVIEW

**Review by Jedimaster4559 — APPROVED** [review]
Looks good @computablee ! Thanks for the update! If you have a moment, please edit the pull request to include some more detailed information about your updates!

I'll merge this once I get reviews from some other people in the leadership team!

**Review by Duke02 — APPROVED** [review]

**Review by sel3450 — APPROVED** [review]

**Pull request merged**
